### PR TITLE
feat: implements institution-partnerships

### PR DESCRIPTION
**Pull request by dzucconi**
The type of this PR is: **Feature**

Implements the /institution-partnerships page. Retires the old page and deletes the `partnerships` app.

### Description

localhost_5050_institution-partnerships2 (3)

### CODE REVIEW

**Comment by dzucconi** [comment]
Some design notes:

* We actually are moving away from the all caps `xs` size. I need to do a sweep of the whole site for this.
* The partner cells use our generic partner cell component; so those look different from the Figma.

**Comment by dzucconi** [comment]
Oh, I missed this bit



Will follow up with that.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/integration` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1b9de8699d1812128e110d6ac960e2b3f1a757fd
</p>

File: src/v2/Apps/InstitutionPartnerships/institutionPartnershipsRoutes.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import loadable from "@loadable/component"
+import { AppRouteConfig } from "v2/System/Router/Route"
+
+// TODO: Bundle not loading
+// /* webpackChunkName: "institutionPartnershipsBundle" */ "./InstitutionPartnershipsApp"
```
Comment (dzucconi) [inline src/v2/Apps/InstitutionPartnerships/institutionPartnershipsRoutes.tsx:5]: Will look at this on Monday. Not sure why this was erroring for me.

**Comment by dzucconi** [comment]
Well... once again `Merge On Green` merged on yellow. This should be fine so...

**Pull request merged**